src/screens/Home/Home.tsx

```diff
@@ -35,6 +71,10 @@ const Home: FC = () => {
             })
           }
         />
+
+        {/* if user === undefined, then we don't know yet. */}
```

Comment:
I think if we have a loading state, then discriminating between `undefined` and `null` shouldn’t be necessary.

Reply:
Since the app is fully usable whether the user is logged in or not, I don't think we should delay the whole app visibility just to know.

This `undefined` does not mean "we have started the signin but its not done". This only happen at app initialization, when we check "is the user already signed in or not".